### PR TITLE
Edit chapter 12

### PR DESCRIPTION
**Pull request by edilmedeiros**
Minor edits, but take a separate look at 6e9bdc271bed7482a846e3e96f1785e842b3c739.

It renames the variable in the code to match the explanations about transaction inputs in the beginning of the text.

### CODE REVIEW

File: 12_reading_inputs_and_type_coercion.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ A description of what the sequence number represents can be found in Mastering B
     for _ in 0..input_length {
         let txid = read_txid(&mut bytes_slice);
         let output_index = read_u32(&mut bytes_slice);
-        let script = read_script(&mut bytes_slice);
+        let script_sig = read_script(&mut bytes_slice);
```
Comment (sb1752) [inline 12_reading_inputs_and_type_coercion.md]: Hm, if we change this here, we'll want to change the `Input` struct field name in chapters 13-22 as well and we'll have to make this change to all the code snapshot files. 

I think it makes sense to remove this from this PR and make a separate PR with this renaming for all chapters after merging in your other changes for chapters 12 and 13. 

I'm happy to do that.

Comment (edilmedeiros) [reply 12_reading_inputs_and_type_coercion.md]: I'll do that and open an issue so we remember that later.

**Comment by edilmedeiros** [comment]
Dropped the `script` variable renaming in favor of doing it later (see #39).

**Comment by sb1752** [comment]
Ack e95b2ffe1255451db0bf8a072adb8db40de23b73

Great thanks
